app/helpers/drawings_helper.rb

```diff
@@ -12,30 +12,41 @@ def form_title
   end
 
   def radio_statuses
-    # Example output: [ ["status1", "status1"], ["status2", "status2"] ]
-    [].tap do |arr|
-      Drawing.statuses.keys.each { |s| arr << [s, s] }
-    end
+    get_humanized_options(Drawing.statuses.keys)
   end
 
   def radio_genders
-    # Example output: [ ["gender0", "Gender 0"], ["gender1", "Gender 1"] ... ]
-    [].tap do |arr|
-      Drawing.genders.keys.each { |s| arr << [s, s.humanize] }
-    end
+    get_humanized_options(Drawing.genders.keys)
   end
 
   def mood_select_box
     # Example output: [ ["1 (sad face)", 1], ["2", 2] ... ["4", 4] ["5 (happy face)", 5] ]
-    selections = [].tap do |arr|
-      (1..5).each { |n| arr << [n.to_s, n] }
+    selections = (1..5).map do |number| 
+      [number.to_s, number]
     end
+
     selections[0][0] += " 😢"
     selections[-1][0] += " 😃"
     selections
   end
 
+  def journey_options
+    get_humanized_options(Drawing::STAGES_OF_JOURNEY)
+  end
+
+  def subject_matter_options
+    get_humanized_options(Drawing::SUBJECT_MATTERS)
+  end
+
   def image_consent_default
     @drawing.new_record? ? true : @drawing.image_consent
   end
+
+  def get_humanized_options(options)
+    # Example output: [ ["gender0", "Gender 0"], ["gender1", "Gender 1"] ... ]
```

Comment:
Yep, this comment can switch around now @echesters as the changes modify the format for this now.